pkg/cloudprovider/instancetypes.go

```diff
@@ -187,7 +187,8 @@ func (p *InstanceTypeProvider) getInstanceTypeZones(ctx context.Context, provide
 		return nil, fmt.Errorf("describing instance type zone offerings, %w", err)
 	}
 	if p.cm.HasChanged("zonal-offerings", provider.SubnetSelector) {
-		logging.FromContext(ctx).Debugf("Discovered EC2 instance types zonal offerings for subnets %s", pretty.Concise(provider.SubnetSelector))
+		ctx = logging.WithLogger(ctx, logging.FromContext(ctx).With("subnetSelector", pretty.Concise(provider.SubnetSelector)))
```

Comment:
I *think* we might be using `kabob-case` rather than `camelCase` in other log contexts. Can you verify?